etc/kate.profile

```diff
@@ -7,19 +7,21 @@ include /etc/firejail/globals.local
 
 # blacklist /run/user/*/bus
 
-noblacklist ${HOME}/.config/katepartrc
-noblacklist ${HOME}/.config/katerc
-noblacklist ${HOME}/.config/kateschemarc
-noblacklist ${HOME}/.config/katesyntaxhighlightingrc
-noblacklist ${HOME}/.config/katevirc
-noblacklist ${HOME}/.local/share/kate
-
-include /etc/firejail/disable-common.inc
+# warning: full filesystem access is granted, allows system administration. Uncomment below  if you want.
+
+# noblacklist ${HOME}/.config/katepartrc
+# noblacklist ${HOME}/.config/katerc
+# noblacklist ${HOME}/.config/kateschemarc
+# noblacklist ${HOME}/.config/katesyntaxhighlightingrc
+# noblacklist ${HOME}/.config/katevirc
+# noblacklist ${HOME}/.local/share/kate
+
+# include /etc/firejail/disable-common.inc
 # include /etc/firejail/disable-devel.inc
-include /etc/firejail/disable-passwdmgr.inc
-include /etc/firejail/disable-programs.inc
+# include /etc/firejail/disable-passwdmgr.inc
```

Comment:
aren't most password manager databases non user editable?

Reply:
We blacklists whole config directories. There may be editable text files.